src/actions/settleUniswapV3Migration.ts

```diff
@@ -35,6 +37,14 @@ export const settleUniswapV3Migration = async ({
   const settlerFeesInBps = BigInt(protocolShareBps) + BigInt(externalParams.senderShareBps || 0);
 
   if (routes.length === 1) {
+    if (
+      JSBI.equal(pool.liquidity, JSBI.BigInt(0)) &&
+      pool.tickCurrent < externalParams.tickUpper &&
+      pool.tickCurrent >= externalParams.tickLower
+    ) {
+      throw new Error('No liquidity for required swap in destination pool');
```

Comment:
Is this accurate? In theory you could have no liquidity on the current tick but still be able to trade on the next tick?